ci/image/Dockerfile.nodejs-centos7-fips-test

```diff
@@ -35,29 +35,49 @@ SHELL [ "/usr/bin/scl", "enable", "devtoolset-8"]
 
 # node-fips environment variables
 ENV NODE_HOME $HOME/node
-ENV NODEJS_VERSION 14.0.0
-ENV FIPSDIR $HOME/install-openssl-fips
-ENV OPENSSL_VERSION 2.0.16
+ENV NODEJS_VERSION 18.0.0
```

Comment:
1. should we include information about version 18 in docker image?
2. why don't we use latest available 18.x version?

Reply:
1. What kind of information would you like to include?
2. I was only able to build `18.0.0` in my testing so far. I saw some [discussion](https://github.com/nodejs/node/issues/43246#issuecomment-1142104510) about v18 not officially supporting centos7 so that could be why building the latest 18.x on the docker had issues

Reply:
ad 1. in the docker image name e.g. `nodejs-centos7-node18-fips-test`
ad 2. It's really strange, thanks for the information